src/ripple/basics/impl/mulDiv.cpp

```diff
@@ -38,9 +38,9 @@ mulDiv(std::uint64_t value, std::uint64_t mul, std::uint64_t div)
     auto constexpr limit = std::numeric_limits<std::uint64_t>::max();
```

Comment:
Maybe move this line in the header file as:

`auto constexpr muldiv_max = std::numeric_limits<std::uint64_t>::max();`

and use that instead of the verbose expression in the other files.